### PR TITLE
Konstantinos / [FE Deriv.com squad] [Office] Add Paris & Guernsey in Contact Us page

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [x] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/contact_us/_offices.tsx [hunk 1]
```diff
@@ -196,7 +220,53 @@ export const Offices = () => {
                     <ResHeader as="h2" type="page-title">
                         {localize('Our offices')}
                     </ResHeader>
+                    <ContinentHeader>{localize('EUROPE')}</ContinentHeader>
                     <GridLayout>
+                        <Flex fd="column" max_width="48.6rem" id="france">
+                            <OfficeHeader>
+                                <div>
+                                    <img src={France} alt="" />
```
Comment (michio-rework) [inline src/pages/contact_us/_offices.tsx:228]: Please provide a value for `alt`, will affect our `SEO` scores. thanks :pray:

File: src/pages/contact_us/_offices.tsx [hunk 2]
```diff
@@ -244,52 +313,83 @@ export const Offices = () => {
                                 </LocalizedLinkText>
                             </BorderBox>
                         </Flex>
-                        <Flex fd="column" max_width="48.6rem" id="dubai">
+                        <Flex fd="column" max_width="48.6rem" id="cyprus">
                             <OfficeHeader>
                                 <div>
-                                    <img src={Dubai} alt="" />
+                                    <img src={Cyprus} alt="" />
```
Comment (michio-rework) [inline src/pages/contact_us/_offices.tsx:319]: Please provide value for `alt` thanks, :pray:

**Review by michio-rework — COMMENTED** [review]
Please provide value for `alt` attributes for all images, it would beneficial for `SEO` score.

File: src/pages/contact_us/_offices.tsx [hunk 3]
```diff
@@ -196,17 +220,62 @@ export const Offices = () => {
...
+                                        </StyledMobileAddress>
+                                    </Mobile>
+                                </LocalizedLinkText>
+                            </BorderBox>
+                        </Flex>
                         <Flex fd="column" max_width="48.6rem" id="malta">
                             <OfficeHeader>
                                 <div>
-                                    <img src={Malta} alt="" />
+                                    <img src={Malta} alt="Malta Icon" />
```
Comment (habib-deriv) [inline src/pages/contact_us/_offices.tsx:273]: The alt test does not look good
Please follow this PR's alt naming convention. 
https://github.com/binary-com/deriv-com/pull/3362/files

File: src/pages/contact_us/_offices.tsx [hunk 4]
```diff
@@ -196,17 +220,62 @@ export const Offices = () => {
...
+                            <BorderBox>
+                                <MapContainer>
+                                    <LocalizedLink
+                                        to="https://g.page/wojo-madeleine-coworking-paris-8?share"
+                                        external="true"
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                    >
+                                        <QueryImage
+                                            alt="Deriv's office location map in France"
```
Comment (habib-deriv) [inline src/pages/contact_us/_offices.tsx:244]: `Deriv's office location map in France` or` Deriv's office location in France` ?

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**